src/amo/components/SearchForm.js

```diff
@@ -102,10 +158,12 @@ export function mapStateToProps(state) {
     addonType: validAddonTypes.includes(state.viewContext.context) ?
       state.viewContext.context : null,
     api: state.api,
+    suggestions: state.autocomplete.suggestions,
   };
 }
 
 export default compose(
+  withErrorHandler({ name: 'Autocomplete' }),
```

Comment:
`SearchForm` seems like the right name; I think using the component's name (the same name we use for the file) is our usual pattern.